test-tracee-ebpf target fails in arm64

$ make -f Makefile.one test-tracee-ebpf

```
make -f Makefile.one ./dist/btfhub
GOOS=linux CC=clang GOARCH=arm64 CGO_CFLAGS="-I/home/rafaeldtinoco/work/sources/ebpf/tracee/dist/libbpf" CGO_LDFLAGS="-lelf -lz /home/rafaeldtinoco/work/sources/ebpf/tracee/dist/libbpf/libbpf.a" go test \
	-tags core,ebpf \
	-v \
	./tracee-ebpf/... \
	./cmd/tracee-ebpf/...
make[1]: warning: -j4 forced in makefile: resetting jobserver mode.
make[1]: 'dist/btfhub' is up to date.
?   	github.com/aquasecurity/tracee/tracee-ebpf/test/gob	[no test files]
=== RUN   Test_Events
    integration_test.go:216: This is an integration test
--- SKIP: Test_Events (0.00s)
PASS
ok  	github.com/aquasecurity/tracee/tracee-ebpf/test/integration	(cached)
=== RUN   TestPrintUint32IP
--- PASS: TestPrintUint32IP (0.00s)
=== RUN   TestPrint16BytesSliceIP
--- PASS: TestPrint16BytesSliceIP (0.00s)
=== RUN   TestReadArgFromBuff
--- PASS: TestReadArgFromBuff (0.00s)
=== RUN   Test_updateProfile
--- PASS: Test_updateProfile (0.00s)
=== RUN   Test_writeProfilerStats
--- PASS: Test_writeProfilerStats (0.00s)
=== RUN   Test_updateFileSHA
--- PASS: Test_updateFileSHA (0.00s)
=== RUN   TestFetchInitNamespaces
    user_mode_events_test.go:15: This is an integration test that requires root permissions
--- SKIP: TestFetchInitNamespaces (0.00s)
PASS
ok  	github.com/aquasecurity/tracee/tracee-ebpf/tracee	(cached)
=== RUN   TestPrepareFilter
=== RUN   TestPrepareFilter/invalid_operator
=== RUN   TestPrepareFilter/invalid_operator#01
=== RUN   TestPrepareFilter/invalid_operator#02
=== RUN   TestPrepareFilter/invalid_operator#03
=== RUN   TestPrepareFilter/invalid_operator#04
=== RUN   TestPrepareFilter/invalid_filter_type
=== RUN   TestPrepareFilter/invalid_filter_type#01
=== RUN   TestPrepareFilter/invalid_filter_type#02
=== RUN   TestPrepareFilter/invalid_uid
=== RUN   TestPrepareFilter/invalid_uid#01
=== RUN   TestPrepareFilter/invalid_pidns
=== RUN   TestPrepareFilter/invalid_uid#02
=== RUN   TestPrepareFilter/invalid_uid#03
=== RUN   TestPrepareFilter/invalid_mntns
=== RUN   TestPrepareFilter/invalid_uid#04
=== RUN   TestPrepareFilter/invalid_uid#05
=== RUN   TestPrepareFilter/invalid_mntns#01
=== RUN   TestPrepareFilter/invalid_uid#06
=== RUN   TestPrepareFilter/invalid_uid#07
=== RUN   TestPrepareFilter/invalid_uid#08
=== RUN   TestPrepareFilter/invalid_uid#09
=== RUN   TestPrepareFilter/invalid_uid#10
=== RUN   TestPrepareFilter/invalid_pidns#01
=== RUN   TestPrepareFilter/invalid_argfilter_1
=== RUN   TestPrepareFilter/invalid_argfilter_2
=== RUN   TestPrepareFilter/invalid_argfilter_3
=== RUN   TestPrepareFilter/invalid_argfilter_4
=== RUN   TestPrepareFilter/invalid_retfilter_1
=== RUN   TestPrepareFilter/invalid_retfilter_2
=== RUN   TestPrepareFilter/invalid_wildcard
=== RUN   TestPrepareFilter/invalid_wildcard_2
=== RUN   TestPrepareFilter/invalid_not_wildcard
=== RUN   TestPrepareFilter/invalid_not_wildcard_2
=== RUN   TestPrepareFilter/uid=0
=== RUN   TestPrepareFilter/uid!=0
=== RUN   TestPrepareFilter/mntns=0
=== RUN   TestPrepareFilter/pidns!=0
=== RUN   TestPrepareFilter/comm=ls
=== RUN   TestPrepareFilter/uts!=deadbeaf
=== RUN   TestPrepareFilter/uid>0
=== RUN   TestPrepareFilter/uid<0
=== RUN   TestPrepareFilter/container
=== RUN   TestPrepareFilter/container=new
=== RUN   TestPrepareFilter/argfilter
    main_test.go:1056:
        	Error Trace:	main_test.go:1056
        	Error:      	Not equal:
        	            	expected: &tracee.ArgFilter{Filters:map[int32]map[string]tracee.ArgFilterVal{257:map[string]tracee.ArgFilterVal{"pathname":tracee.ArgFilterVal{Equal:[]string{"/bin/ls", "/tmp/tracee"}, NotEqual:[]string{"/etc/passwd"}}}}, Enabled:true}
        	            	actual  : &tracee.ArgFilter{Filters:map[int32]map[string]tracee.ArgFilterVal{56:map[string]tracee.ArgFilterVal{"pathname":tracee.ArgFilterVal{Equal:[]string{"/bin/ls", "/tmp/tracee"}, NotEqual:[]string{"/etc/passwd"}}}}, Enabled:true}

        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -2,3 +2,3 @@
        	            	  Filters: (map[int32]map[string]tracee.ArgFilterVal) (len=1) {
        	            	-  (int32) 257: (map[string]tracee.ArgFilterVal) (len=1) {
        	            	+  (int32) 56: (map[string]tracee.ArgFilterVal) (len=1) {
        	            	    (string) (len=8) "pathname": (tracee.ArgFilterVal) {
        	Test:       	TestPrepareFilter/argfilter
=== RUN   TestPrepareFilter/retfilter
    main_test.go:1057:
        	Error Trace:	main_test.go:1057
        	Error:      	Not equal:
        	            	expected: &tracee.RetFilter{Filters:map[int32]tracee.IntFilter{257:tracee.IntFilter{Equal:[]int64{2}, NotEqual:[]int64{}, Greater:1, Less:-9223372036854775808, Is32Bit:false, Enabled:true}}, Enabled:true}
        	            	actual  : &tracee.RetFilter{Filters:map[int32]tracee.IntFilter{56:tracee.IntFilter{Equal:[]int64{2}, NotEqual:[]int64{}, Greater:1, Less:-9223372036854775808, Is32Bit:false, Enabled:true}}, Enabled:true}

        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -2,3 +2,3 @@
        	            	  Filters: (map[int32]tracee.IntFilter) (len=1) {
        	            	-  (int32) 257: (tracee.IntFilter) {
        	            	+  (int32) 56: (tracee.IntFilter) {
        	            	    Equal: ([]int64) (len=1) {
        	Test:       	TestPrepareFilter/retfilter
=== RUN   TestPrepareFilter/wildcard_filter
=== RUN   TestPrepareFilter/wildcard_not_filter
=== RUN   TestPrepareFilter/multiple_filters
--- FAIL: TestPrepareFilter (0.00s)
    --- PASS: TestPrepareFilter/invalid_operator (0.00s)
    --- PASS: TestPrepareFilter/invalid_operator#01 (0.00s)
    --- PASS: TestPrepareFilter/invalid_operator#02 (0.00s)
    --- PASS: TestPrepareFilter/invalid_operator#03 (0.00s)
    --- PASS: TestPrepareFilter/invalid_operator#04 (0.00s)
    --- PASS: TestPrepareFilter/invalid_filter_type (0.00s)
    --- PASS: TestPrepareFilter/invalid_filter_type#01 (0.00s)
    --- PASS: TestPrepareFilter/invalid_filter_type#02 (0.00s)
    --- PASS: TestPrepareFilter/invalid_uid (0.00s)
    --- PASS: TestPrepareFilter/invalid_uid#01 (0.00s)
    --- PASS: TestPrepareFilter/invalid_pidns (0.00s)
    --- PASS: TestPrepareFilter/invalid_uid#02 (0.00s)
    --- PASS: TestPrepareFilter/invalid_uid#03 (0.00s)
    --- PASS: TestPrepareFilter/invalid_mntns (0.00s)
    --- PASS: TestPrepareFilter/invalid_uid#04 (0.00s)
    --- PASS: TestPrepareFilter/invalid_uid#05 (0.00s)
    --- PASS: TestPrepareFilter/invalid_mntns#01 (0.00s)
    --- PASS: TestPrepareFilter/invalid_uid#06 (0.00s)
    --- PASS: TestPrepareFilter/invalid_uid#07 (0.00s)
    --- PASS: TestPrepareFilter/invalid_uid#08 (0.00s)
    --- PASS: TestPrepareFilter/invalid_uid#09 (0.00s)
    --- PASS: TestPrepareFilter/invalid_uid#10 (0.00s)
    --- PASS: TestPrepareFilter/invalid_pidns#01 (0.00s)
    --- PASS: TestPrepareFilter/invalid_argfilter_1 (0.00s)
    --- PASS: TestPrepareFilter/invalid_argfilter_2 (0.00s)
    --- PASS: TestPrepareFilter/invalid_argfilter_3 (0.00s)
    --- PASS: TestPrepareFilter/invalid_argfilter_4 (0.00s)
    --- PASS: TestPrepareFilter/invalid_retfilter_1 (0.00s)
    --- PASS: TestPrepareFilter/invalid_retfilter_2 (0.00s)
    --- PASS: TestPrepareFilter/invalid_wildcard (0.00s)
    --- PASS: TestPrepareFilter/invalid_wildcard_2 (0.00s)
    --- PASS: TestPrepareFilter/invalid_not_wildcard (0.00s)
    --- PASS: TestPrepareFilter/invalid_not_wildcard_2 (0.00s)
    --- PASS: TestPrepareFilter/uid=0 (0.00s)
    --- PASS: TestPrepareFilter/uid!=0 (0.00s)
    --- PASS: TestPrepareFilter/mntns=0 (0.00s)
    --- PASS: TestPrepareFilter/pidns!=0 (0.00s)
    --- PASS: TestPrepareFilter/comm=ls (0.00s)
    --- PASS: TestPrepareFilter/uts!=deadbeaf (0.00s)
    --- PASS: TestPrepareFilter/uid>0 (0.00s)
    --- PASS: TestPrepareFilter/uid<0 (0.00s)
    --- PASS: TestPrepareFilter/container (0.00s)
    --- PASS: TestPrepareFilter/container=new (0.00s)
    --- FAIL: TestPrepareFilter/argfilter (0.00s)
    --- FAIL: TestPrepareFilter/retfilter (0.00s)
    --- PASS: TestPrepareFilter/wildcard_filter (0.00s)
    --- PASS: TestPrepareFilter/wildcard_not_filter (0.00s)
    --- PASS: TestPrepareFilter/multiple_filters (0.00s)
=== RUN   TestPrepareCapture
=== RUN   TestPrepareCapture/various_capture_options
=== RUN   TestPrepareCapture/various_capture_options/invalid_capture_option
=== RUN   TestPrepareCapture/various_capture_options/invalid_capture_dir
=== RUN   TestPrepareCapture/various_capture_options/invalid_network_interface
=== RUN   TestPrepareCapture/various_capture_options/invalid_capture_write_filter
=== RUN   TestPrepareCapture/various_capture_options/invalid_capture_write_filter_2
=== RUN   TestPrepareCapture/various_capture_options/empty_capture_write_filter
=== RUN   TestPrepareCapture/various_capture_options/capture_mem
=== RUN   TestPrepareCapture/various_capture_options/capture_exec
=== RUN   TestPrepareCapture/various_capture_options/capture_module
=== RUN   TestPrepareCapture/various_capture_options/capture_write
=== RUN   TestPrepareCapture/various_capture_options/capture_write_filtered
=== RUN   TestPrepareCapture/various_capture_options/multiple_capture_options
=== RUN   TestPrepareCapture/various_capture_options/capture_exec_and_enable_profile
=== RUN   TestPrepareCapture/various_capture_options/just_enable_profile
=== RUN   TestPrepareCapture/various_capture_options/network_interface
=== RUN   TestPrepareCapture/various_capture_options/network_interface,_but_repeated
=== RUN   TestPrepareCapture/clear_dir
--- PASS: TestPrepareCapture (0.00s)
    --- PASS: TestPrepareCapture/various_capture_options (0.00s)
        --- PASS: TestPrepareCapture/various_capture_options/invalid_capture_option (0.00s)
        --- PASS: TestPrepareCapture/various_capture_options/invalid_capture_dir (0.00s)
        --- PASS: TestPrepareCapture/various_capture_options/invalid_network_interface (0.00s)
        --- PASS: TestPrepareCapture/various_capture_options/invalid_capture_write_filter (0.00s)
        --- PASS: TestPrepareCapture/various_capture_options/invalid_capture_write_filter_2 (0.00s)
        --- PASS: TestPrepareCapture/various_capture_options/empty_capture_write_filter (0.00s)
        --- PASS: TestPrepareCapture/various_capture_options/capture_mem (0.00s)
        --- PASS: TestPrepareCapture/various_capture_options/capture_exec (0.00s)
        --- PASS: TestPrepareCapture/various_capture_options/capture_module (0.00s)
        --- PASS: TestPrepareCapture/various_capture_options/capture_write (0.00s)
        --- PASS: TestPrepareCapture/various_capture_options/capture_write_filtered (0.00s)
        --- PASS: TestPrepareCapture/various_capture_options/multiple_capture_options (0.00s)
        --- PASS: TestPrepareCapture/various_capture_options/capture_exec_and_enable_profile (0.00s)
        --- PASS: TestPrepareCapture/various_capture_options/just_enable_profile (0.00s)
        --- PASS: TestPrepareCapture/various_capture_options/network_interface (0.00s)
        --- PASS: TestPrepareCapture/various_capture_options/network_interface,_but_repeated (0.00s)
    --- PASS: TestPrepareCapture/clear_dir (0.00s)
=== RUN   TestPrepareOutput
=== RUN   TestPrepareOutput/invalid_output_option
=== RUN   TestPrepareOutput/invalid_output_option#01
=== RUN   TestPrepareOutput/invalid_output_option_2
=== RUN   TestPrepareOutput/empty_val
=== RUN   TestPrepareOutput/option_stack-addresses
=== RUN   TestPrepareOutput/option_detect-syscall
=== RUN   TestPrepareOutput/option_exec-env
=== RUN   TestPrepareOutput/option_exec-hash
=== RUN   TestPrepareOutput/all_options
--- PASS: TestPrepareOutput (0.00s)
    --- PASS: TestPrepareOutput/invalid_output_option (0.00s)
    --- PASS: TestPrepareOutput/invalid_output_option#01 (0.00s)
    --- PASS: TestPrepareOutput/invalid_output_option_2 (0.00s)
    --- PASS: TestPrepareOutput/empty_val (0.00s)
    --- PASS: TestPrepareOutput/option_stack-addresses (0.00s)
    --- PASS: TestPrepareOutput/option_detect-syscall (0.00s)
    --- PASS: TestPrepareOutput/option_exec-env (0.00s)
    --- PASS: TestPrepareOutput/option_exec-hash (0.00s)
    --- PASS: TestPrepareOutput/all_options (0.00s)
=== RUN   Test_checkCommandIsHelp
=== RUN   Test_checkCommandIsHelp/no_flag
=== RUN   Test_checkCommandIsHelp/help_flag
=== RUN   Test_checkCommandIsHelp/capture_flag
=== RUN   Test_checkCommandIsHelp/output_flag
=== RUN   Test_checkCommandIsHelp/trace_flag
=== RUN   Test_checkCommandIsHelp/multiple_flags
--- PASS: Test_checkCommandIsHelp (0.00s)
    --- PASS: Test_checkCommandIsHelp/no_flag (0.00s)
    --- PASS: Test_checkCommandIsHelp/help_flag (0.00s)
    --- PASS: Test_checkCommandIsHelp/capture_flag (0.00s)
    --- PASS: Test_checkCommandIsHelp/output_flag (0.00s)
    --- PASS: Test_checkCommandIsHelp/trace_flag (0.00s)
    --- PASS: Test_checkCommandIsHelp/multiple_flags (0.00s)
=== RUN   Test_checkClangVersion
--- PASS: Test_checkClangVersion (0.00s)
=== RUN   Test_checkRequiredCapabilites
=== RUN   Test_checkRequiredCapabilites/happy_path
=== RUN   Test_checkRequiredCapabilites/CAP_SYS_ADMIN_missing
=== RUN   Test_checkRequiredCapabilites/CAP_IPC_LOCK_missing
--- PASS: Test_checkRequiredCapabilites (0.00s)
    --- PASS: Test_checkRequiredCapabilites/happy_path (0.00s)
    --- PASS: Test_checkRequiredCapabilites/CAP_SYS_ADMIN_missing (0.00s)
    --- PASS: Test_checkRequiredCapabilites/CAP_IPC_LOCK_missing (0.00s)
=== RUN   Test_fetchFormattedEventParams
--- PASS: Test_fetchFormattedEventParams (0.00s)
FAIL
FAIL	github.com/aquasecurity/tracee/cmd/tracee-ebpf	0.009s
FAIL
make: *** [Makefile.one:401: test-tracee-ebpf] Error 1
```

--- a/cmd/tracee-ebpf/main_test.go
+++ b/cmd/tracee-ebpf/main_test.go
@@ -796,7 +796,7 @@ func TestPrepareFilter(t *testing.T) {
 				NewContFilter: &tracee.BoolFilter{},
 				ArgFilter: &tracee.ArgFilter{
 					Filters: map[int32]map[string]tracee.ArgFilterVal{
-						257: {
+						tracee.OpenatEventID: {
 							"pathname": tracee.ArgFilterVal{
 								Equal:    []string{"/bin/ls", "/tmp/tracee"},
 								NotEqual: []string{"/etc/passwd"},
@@ -857,7 +857,7 @@ func TestPrepareFilter(t *testing.T) {
 				},
 				RetFilter: &tracee.RetFilter{
 					Filters: map[int32]tracee.IntFilter{
-						257: {
+						tracee.OpenatEventID: {
 							Equal:    []int64{2},
 							NotEqual: []int64{},
 							Less:     tracee.LessNotSetInt,
@@ -874,7 +874,7 @@ func TestPrepareFilter(t *testing.T) {
 			testName: "wildcard filter",
 			filters:  []string{"event=open*"},
 			expectedFilter: tracee.Filter{
-				EventsToTrace: []int32{2, 257},
+				EventsToTrace: []int32{tracee.OpenEventID, tracee.OpenatEventID},
 				UIDFilter: &tracee.UintFilter{
 					Equal:    []uint64{},
 					NotEqual: []uint64{},

